css-fonts-5/Overview.bs

```diff
@@ -59,7 +60,78 @@ rules defined in CSS Fonts Level 4.
 This specification is currently a delta to the CSS Fonts Level 4 specification.
 Do not assume that if something is not here, it has been dropped.
 
+<h2 id="text-scale-meta">
+Text-Scale <code class=html>&lt;meta&gt;</code> element</h2>
+
+A document with a <{meta}> tag
+whose <code>name</code> attribute
+is a <a>ASCII case-insensitive</a> match for
+<dfn lt=text-scale><code>"text-scale"</code></dfn>
+is recognized as setting the computed value
+of the ''font-size/medium'' font size
+and consequently scaling the computed size of the other <<absolute-size>> keywords.
+
+The value of the <code>content</code>
+attribute must be an <a>ASCII case-insensitive</a> match for one of the
+recognized keywords. Otherwise the tag is ignored.
+
+Documents without this <code>&lt;meta></code> tag will have an assumed default
+value of <code>legacy</code>.
+
```

Comment:
It would be good to add an example block here, [similar to the example in the HTML spec for the `color-scheme` meta tag.](https://html.spec.whatwg.org/multipage/semantics.html#meta-color-scheme)